setup.py

```diff
@@ -19,6 +19,7 @@ def find_scripts(script_dir, prefix):
 
 
 lstchain_list = find_scripts("lstchain/scripts", "lstchain_")
+lstchain_list += find_scripts("lstchain/scripts", "longterm_")
```

Comment:
I'd rather add the `lstchain` prefix to the long term script. Any reason that was not there?

Reply:
No, no good reason. Will do, please @morcuended keep in mind it will be lstchain_longterm_dl1_check.py in the next release.

Reply:
Thank you. This will make things easier.